kurtosis-devnet/cmd/main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/build"
+	"github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/kurtosis"
+	"github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/serve"
+	"github.com/ethereum-optimism/optimism/kurtosis-devnet/pkg/tmpl"
+)
+
+type config struct {
+	templateFile    string
+	dataFile        string
+	kurtosisPackage string
+	enclave         string
+	environment     string
+	dryRun          bool
+	localHostName   string
+	baseDir         string
+}
+
+func parseFlags(fs *flag.FlagSet, args []string) (*config, error) {
+	cfg := &config{}
+
+	fs.StringVar(&cfg.templateFile, "template", "", "Path to the template file (required)")
```

Comment:
[nit] Can you use urfave for this? It's what we've used everywhere else.

Reply:
sure, done !